servlet.py

```diff
@@ -251,17 +256,15 @@ def get(self):
         query = self.get_argument('q', default='pairs')
 
         if query == 'pairs':
-            responseData = []
-            for pair in self.pairs:
-                (l1, l2) = pair.split('-')
-                responseData.append({'sourceLanguage': l1, 'targetLanguage': l2})
-                if self.get_arguments('include_deprecated_codes'):
-                    responseData.append({'sourceLanguage': toAlpha2Code(l1), 'targetLanguage': toAlpha2Code(l2)})
-            self.sendResponse({'responseData': responseData, 'responseDetails': None, 'responseStatus': 200})
-        elif query == 'chains':
-            src = self.get_argument('src', default=None)
+            src = self.get_argument('chainSource', default=None)
```

Comment:
Use `src` because this function should output the correct results whether chaining is enabled or not. If chaining is enabled, it'll just have more results.